app/javascript/angular/code/services/_fixed_sessions.js

```diff
@@ -236,17 +213,14 @@ export const fixedSessions = (
 
       const data = params.get("data");
 
-      // _.values suggests that `params.get('selectedSessionIds')` could be an obj, is it true?
-      const sessionIds = _.values(params.get("selectedSessionIds") || []);
-
       if (!data.timeFrom || !data.timeTo) return;
 
       var reqData = {
         time_from: data.timeFrom,
         time_to: data.timeTo,
         tags: data.tags,
         usernames: data.usernames,
-        session_ids: sessionIds
```

Comment:
maybe we should also change that endpoint, to only expect one id, but I'll leave this for the future